.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Overview
+
+Brief description of what this PR does, and why it is needed.
+
+### Checklist
+
+- [ ] Description of PR is in an appropriate section of the [changelog](https://github.com/raster-foundry/raster-foundry/blob/develop/CHANGELOG.md) and grouped with similar changes if possible
+- [ ] New tables and queries have appropriate indices added
+- [ ] Any new migrations have been audited to make sure they won't kill the application while being applied
+- [ ] Any new API endpoints have tests
+- [ ] If there are breaking API changes, you've hidden them behind a new API version
```

Comment:
It seems kind of premature to add this requirement at this point -- at least until this is deployed or considered "production" -- we're going to be alpha/beta-ish for a bit.

Reply:
That's a pretty fair quibble